pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -206,7 +233,10 @@ func (og *operationGenerator) GenerateVolumesAreAttachedFunc(
 						volumeSpecMap[spec], spec.Name(), nodeName)
 				}
 			}
+
+			klog.Infof("VerifyVolumesAreAttached completed for plugin %s", pluginName)
```

Comment:
IMO this is too verbose.

Reply:
changed to V(4). The operation generator is not very consistent in logging verbosity in general, some `1,2,4` scattered around. Some calls have full verbosity for success calls, some dont log success at any verbosity at all :)

A fix for another day.